pycytominer/tests/test_annotate.py

```diff
@@ -70,6 +70,32 @@ def test_annotate_platemap_naming():
     pd.testing.assert_frame_equal(result, expected_result)
 
 
+def test_annotate_merge():
+    # Test to ensure that the "_platemap" merge suffix is applied to the platemap columns when there is a name collision
+    platemap_modified_df = PLATEMAP_DF.copy(deep=True)
+    platemap_modified_df["x"] = [1, 2, 3, 4, 5, 6]
+
+    expected_result = platemap_modified_df.merge(
+        DATA_DF,
+        left_on="well_position",
+        right_on="Metadata_Well",
+        suffixes=("_platemap", None),
+    ).drop("well_position", axis="columns")
+    # Reorder with Metadtata column first
+    meta_cols = ["Metadata_Well"]
+    other_cols = expected_result.drop(meta_cols, axis="columns").columns.tolist()
+    expected_result = expected_result.loc[:, meta_cols + other_cols]
```

Comment:
I copied the column re-arranging logic from `annotate.py` and think it would be better to stick to that logic unless we are making changes in both places.

Reply:
I changed to an explicit setting of column order which seems like the simplest solution that will also alert us if something unintentionally changes the way pycytominer sorts columns.

Reply:
this looks good to me